src/loaders/thematicLoader.js

```diff
@@ -70,7 +70,7 @@ const thematicLoader = async config => {
         title: name,
         period: period
             ? names[data.metaData.dimensions.pe[0]]
-            : `${formatLocaleDatee(config.startDate)} - ${formatLocaleDate(
+            : `${formatLocaleDate(config.startDate)} - ${formatLocaleDate(
```

Comment:
YAY LINTING!

Reply:
Need to backport this fix to 2.31. Will do.

Reply:
Fix backported to 2.31